docs/source/workflows/design_workflows.rst

```diff
@@ -182,6 +187,10 @@ A branch has a name, along with any number of design workflows. A DesignWorkflow
 
 When you're done with a branch, it can be archived, removing it from the results of ``list`` and setting the ``archived`` flag.
 ``list_archived`` lists all archived branches in a project. An archived branch can be restored via its unique ID.
+Note that archiving branches are independent of archiving the design workflows contained within it.
+Archiving a branch, by removing it from listing methods, will hide the entire branch from default displays in the web UI.
```

Comment:
It may be worth mentioning here that branches are the only place where design workflows are displayed in the UI. One point that is not fully addressed here is that while design workflows are not archived when a parent branch is archived, the design workflows are no longer visible in the UI. This was a major point of confusion for me initially, and I think it would be helpful to call this behavior out explicitly.

Reply:
+1 on Will's comment, and wanted to add that the reverse behavior is also confusing for (probably) the same reason - if I archive all the workflows in a branch, the branch will still exist but won't be shown on the UI

Reply:
Let me know if the most recent version makes the relationship more clear. I would like to avoid being too specific about the behavior of our UI, given that it can change independently of our SDK.